native-image-workshop/README.md

```diff
@@ -124,9 +134,8 @@ Compare that to the time running the app with the regular `java` command:
 time java -cp ./target/graalvmnidemos-1.0-SNAPSHOT-jar-with-dependencies.jar oracle.App
 ```
 
-What do the various parameters we passed to the `native-image` command do? Full documentation on these can be found [here](https://www.graalvm.org/docs/reference-manual/native-image/#image-generation-options):
+What do the various parameters we passed to the `native-image` command do? Full documentation on these can be found [here](https://www.graalvm.org/reference-manual/native-image/BuildConfiguration/):
 
-* `--no-server`: Do not start a build server process. For these examples we just want to run the builds. It is possible to run the builds through a build server.
 * `--no-fallback`: Do not generate a fallback image. A fallback image requires the JVM to run, and we do not want this.We just want it to be a native image.
```

Comment:
Add a space between  _... want this.We just want ..._

Reply:
updated